src/ServiceControl.Monitoring/MetricReport.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace NServiceBus.Metrics
+{
+    using Newtonsoft.Json.Linq;
+
+    /// <summary>
+    /// The reporting message.
+    /// </summary>
+    public class MetricReport : ICommand
```

Comment:
After taking a look how the raw data were reported with `NServiceBusReceivedMetricContext` I think that I agree with @weralabaj . The reporting could be done as a [TolerantReader](https://martinfowler.com/bliki/TolerantReader.html) that simply extracts `JProperty` with name `Context` and store the whole JObject payload in a dictionary. This would remove our dependency on the Metrics leaving this data as raw as possible.